### DISCUSSION TITLE
Add Local Timestep + Local CFL as Volume Output?

### DISCUSSION THREAD

**Discussion by j-signorelli** [su2code/SU2#2056] [Ideas]
Doesn't look like SU2 is capable of this presently, but it would be nice to be able to view, for example, local timestep for each cell for steady-state volume outputs to get a better idea of what regions of the flowfield are driving small timesteps, or local CFL for unsteady dual-timestepping ones.

**Comment by bigfooted** [comment]
Hi, 

We don't output it, but I also think this could be useful/insightful. Can you make an issue (feature request) for it? Can you implement this yourself?

**Reply by j-signorelli** [reply]
Got it, PR submitted: #2060
